lib/deploy_service.rb

```diff
@@ -60,7 +60,7 @@ def get_dev_version(configuration)
   def delete_ui_objects(service_id, version_number)
     # Delete objects created by the UI. We want VCL to be the source of truth.
     # Most of these don't have real objects in the Fastly API gem.
-    to_delete = %w[healthcheck cache_settings request_settings response_object header gzip]
+    to_delete = %w[healthcheck cache_settings request_settings response_object header]
```

Comment:
Something tells me the length of this array is going to continue to get smaller and smaller 😆